src/plugins/intel_gpu/thirdparty/CMakeLists.txt

```diff
@@ -47,7 +47,9 @@ if(ENABLE_ONEDNN_FOR_GPU)
 
         foreach(cmake_var IN ITEMS CMAKE_SYSTEM_NAME CMAKE_SYSTEM_VERSION
                                    CMAKE_SYSTEM_PROCESSOR CMAKE_TOOLCHAIN_FILE)
-            list(APPEND cmake_extra_args "-D${cmake_var}=${${cmake_var}}")
```

Comment:
Is there any reason to delete this line?

Reply:
updated, it was a typo.
thank you for catching